apps/etterlatte-saksbehandling-ui/client/src/components/oppgavebenk/GosysOppgaveliste.tsx

```diff
@@ -35,7 +40,25 @@ export const GosysOppgaveliste = ({ saksbehandlereIEnhet }: Props) => {
     return <Tilgangsmelding />
   }
 
-  const [filter, setFilter] = useState<GosysFilter>(hentGosysOppgaverFilterFraLocalStorage())
+  const harBareEnLokalSaksbehandlendeEnhet = (enheter: string[]): string[] => {
+    const lokaleEnheter = [ENHETFILTER.E4815, ENHETFILTER.E4808, ENHETFILTER.E4817]
+    return enheter.filter((enhet) => lokaleEnheter.includes(enhet))
+  }
+
+  const setLokalEnhetSomStandardOmIngenLocalStorage = (): GosysFilter => {
+    const gosysFilter = hentGosysOppgaverFilterFraLocalStorage()
+    if (!gosysFilter.enhetFilter) {
+      const enheterForSaksbehandler = innloggetSaksbehandler.enheter
+      const lokaleEnheterForSaksbehandler = harBareEnLokalSaksbehandlendeEnhet(enheterForSaksbehandler)
+      if (lokaleEnheterForSaksbehandler.length == 1) {
+        return { ...gosysFilter, enhetFilter: lokaleEnheterForSaksbehandler[0] as EnhetFilterKeys }
+      }
+      return { ...gosysFilter }
+    }
+    return gosysFilter
+  }
+
+  const [filter, setFilter] = useState<GosysFilter>(setLokalEnhetSomStandardOmIngenLocalStorage())
```

Comment:
Putt det heller i fila dedikert til filter i localstorage, tror den heter `filterLocalStorage.ts`

Reply:
Dette har ingenting å gjøre med localStorage men den kunne potensielt vært i en filtrerings komponent

Reply:
Aha, kanskje ha det i en av utils filene for filtere da, er vel mer ren logikk for å lage filteret enn selve visualiseringen av filteret